core/crypto/src/signature.rs

```diff
@@ -443,6 +443,13 @@ impl Signature {
     }
 }
 
+impl Default for Signature {
+    fn default() -> Self {
+        let signature_str = "\"secp256k1:Ng9nnxbC1BBWJ8NZ8XyExB7kUMtWNn8xCdxRtVC1m7ghH8EpwcupZU3BAxk6ehamaB89aSzhC6UeME4QgZtfkhtwi\"";
```

Comment:
that's a weird default, where is this signature coming from?

Reply:
should we just add ::empty() which returns 0s? or what is the purpose of this?

Reply:
Yes sure, this was the result of few minutes trying to hardcode some signature. Didn't find ::empy() 🤦